osquery/core/windows/ntapi.h

```diff
@@ -0,0 +1,108 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#pragma once
+
+
+#include <windows.h>
+namespace osquery {
+namespace tables {
+
+#define NTSTATUS ULONG
+#define STATUS_SUCCESS 0L
+
+#define OBJ_CASE_INSENSITIVE 64L
+#define DIRECTORY_QUERY 0x0001
+#define SYMBOLIC_LINK_QUERY 0x0001
+
+typedef enum _SYSTEM_INFORMATION_CLASS {
+  SystemProcessorInformation,
+  SystemPerformanceInformation,
+  SystemTimeOfDayInformation,
+  SystemPathInformation,
+  SystemProcessInformation
+} SYSTEM_INFORMATION_CLASS;
```

Comment:
As far as I know you could write is in more C++ way, like:
```c++
enum  SYSTEM_INFORMATION_CLASS {
  SystemProcessorInformation,
  SystemPerformanceInformation,
  SystemTimeOfDayInformation,
  SystemPathInformation,
  SystemProcessInformation
};
```
right?

Reply:
@akindyakov the reasoning behind how "ntapi.h" is constructed was to mimic the Microsoft defintions, including the style.  The idea was to be consistent with MS-style declarations in this MS-specific header.  any thoughts on this?

Reply:
My 2-cents -- I'd say let's leave this for consistency, as it's a standard API that one can find in many MS API interfaces. That being said I'd be curious of @akindyakov's thoughts on it

Reply:
There is no big difference, so we can keep it as-is, no problem.

Reply:
@akindyakov appreciated, thanks!